src/Catty/DataModel/Scripts/Script.m

```diff
@@ -47,6 +47,7 @@ - (void)addBrick:(Brick<BrickProtocol>*)brick atIndex:(NSUInteger)index
     CBAssert([self.brickList indexOfObject:brick] == NSNotFound);
     brick.script = self;
     [brick.script.brickList insertObject:brick atIndex:index];
+    [Util updateRecentlyUsedBricksFor:NSStringFromClass(brick.class)];
```

Comment:
Is this line here really necessary? As far as I can see it is enough to update the list in the `BrickCategoryViewController`.